hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/service/InventoryConfig.java

```diff
@@ -91,6 +97,17 @@ public void init() {
                 log.errorInventoryCacheNotFound();
                 throw new IllegalStateException("Inventory query factory for resource_type cache is not found");
             }
+            if (ispnReindex) {
+                log.infoStartInventoryReindex();
+                long startReindex = System.currentTimeMillis();
+                SearchManager searchResourceManager = Search.getSearchManager(resource);
+                searchResourceManager.getMassIndexer().start();
```

Comment:
If it's a time-consuming operation, there may be a benefit to run them in parallel ?

Reply:
Mmm, yes, you are right, as the caches are splitted it might help.
I will add that.

Reply:
As those are separated caches it should, +1, I will add this suggestion.